### DISCUSSION TITLE
Welcome to Sketchware-Pro Discussions!

### DISCUSSION THREAD

**Discussion by hasrat-ali** [Sketchware-Pro/Sketchware-Pro#92] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community, showing off on other members is seriously **not** allowed. we all here to build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hasrat-ali** [comment]
Hi, this is the start of `Discussions`

**Reply by SectionTN** [reply]
hi

**Reply by j-px** [reply]
hi

**Reply by ghost** [reply]
Ded chat xd

**Comment by khaled-0** [comment]
Por

**Comment by inari-codes** [comment]
~~Can I Use Sketchwer Pro To Hack Nasa??~~

**Reply by hasrat-ali** [reply]
Can you just don't

**Reply by inari-codes** [reply]
ok it's just a bad joke
